dice.js

```diff
@@ -0,0 +1,7 @@
+
+const rollDice =function(){
+    let number1 = Math.floor((Math.random() * 6) + 1);
+    let number2 = Math.floor((Math.random() * 6) + 1);
+    let add = number1 + number2;
+    console.log("Dice rolled " + num1 + ", " + num2 + ". Sum is " + add);
```

Comment:
Remember to test your code before submitting it.
the variables you declared were `number1` and `number2` , yet you used `num1` and `num2` in the `console.log()`.